Automattic/newspack-popups
# Add all campaigns view (Campaigns wizard from Newspack Plugin)

### All Submissions:






### Changes proposed in this Pull Request:



Include the enhanced campaigns UI (Campaigns wizard from Newspack Plugin) in this plugin.
The Campaigns wizard is removed from Newspack plugin in https://github.com/Automattic/newspack-plugin/pull/594

### How to test the changes in this Pull Request:

1. In WP Admin, click "All Campaigns", observe the UI from Newspack plugin instead of native WP posts list UI rendered
2. Test out the whole functionality offered by this view, it should work in the same way as in the Newspack Plugin
3. Observe that the logo in the top part of the view links to:
    - Newspack plugin dashboard, if the Newspack plugin is active
    - newspack.pub if the Newspack plugin is not active

### Other information:
